parent/pom.xml

```diff
@@ -1458,6 +1458,10 @@
                            <key>infinispan.isCoreModule</key>
                            <value>${module.metadata.isCoreModule}</value>
                         </systemProperty>
+                        <systemProperty>
+                           <key>log4j.configurationFile</key>
+                           <value>${log4j.configurationFile}</value>
+                        </systemProperty>
```

Comment:
I'm not sure about this, as running the tool could overwrite the test suite's log file... True, the tool should run before the tests, and `infinispan.module-suffix` is not defined here, but I'd prefer having a tools-only log4j2 configuration that just writes to the console.